build.gradle

```diff
@@ -11,7 +11,6 @@ buildscript {
     }
 }
 plugins {
-    id "com.github.hierynomus.license" version "0.12.1"
```

Comment:
I don't see a point to mention the license anymore, because the founder of the library is not active anymore, and the library have been changed a lot since.
But I don't familiar with the rules, so if it's not legal/legitimate, I will put it back

Reply:
I have no idea neither about the license, but I prefer to keep the Apache license header everywhere and keep the plugin to enforce it

Reply:
Put it back